cmd/limactl/genschema.go

```diff
@@ -3,21 +3,27 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"os"
 
+	"github.com/goccy/go-yaml"
 	"github.com/invopop/jsonschema"
 	"github.com/lima-vm/lima/pkg/limayaml"
+	jsonschema2 "github.com/santhosh-tekuri/jsonschema/v6"
```

Comment:
Would it be possible to avoid importing two jsonschema libs

Reply:
One is for generating, one is for validating. Their naming is unfortunate.

https://json-schema.org/tools?query=&languages=Go

Reply:
Their import names could be better than "1" and "2" though.

Perhaps if it was put in two different files, as helper functions?